src/main/java/com/facebook/openwifirrm/ucentral/informationelement/Country.java

```diff
@@ -124,15 +125,8 @@ public static Country parse(JsonObject contents) {
 		JsonElement constraintsObject = contents.get("constraints");
 		if (constraintsObject != null) {
 			for (JsonElement jsonElement : constraintsObject.getAsJsonArray()) {
-				try {
-					constraints
-						.add(CountryInfo.parse(jsonElement.getAsJsonObject()));
-				} catch (NullPointerException e) {
-					logger.debug(
-						"Skipping invalid constraint encountered in Country IE.",
-						e
-					);
-				}
+				CountryInfo countryInfo = CountryInfo.parse(jsonElement.getAsJsonObject());
```

Comment:
Similar to my other comment - a null check for every field instead of catching an exception (which shouldn't happen often) seems like too much extra code? Thoughts?